doc/user/main.xml

```diff
@@ -74,10 +75,10 @@
 
     <corpauthor>Steven Knight and the SCons Development Team</corpauthor>
 
-    <pubdate>2004 - 2019</pubdate>
+    <pubdate>2004 - 2020</pubdate>
```

Comment:
Anyway we can put this in some single file all the docbooks can use?

Reply:
it's just in two files now, so not worth a lot of effort to generalize down to one, IMO.  But yes, copyrightyear should be generated, not hand-edited.